OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/IntentBuilder.java

```diff
@@ -0,0 +1,54 @@
+package com.ooyala.sample.players;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+
+public class IntentBuilder {
+
+  private Class<? extends Activity> activityClass;
```

Comment:
Activity class also, having extra link to Activity may cause leaks

Reply:
Class is just a class object it will not cause any leaks